modules/consensus/synchronize.go

```diff
@@ -64,7 +64,11 @@ func (s *State) receiveBlocks(conn modules.PeerConn) error {
 			s.verificationRigor = fullVerification
 			acceptErr := s.acceptBlock(block)
 			s.mu.Unlock(lockID)
-			if acceptErr != nil && acceptErr != modules.ErrNonExtendingBlock {
+			// these errors are benign
+			if acceptErr == modules.ErrNonExtendingBlock || acceptErr == ErrBlockKnown {
+				acceptErr = nil
+			}
+			if acceptErr != nil {
```

Comment:
Is this the right approach? Don't we want to indicate that we didn't need the blocks?

I guess not.